lib/nori/parser/rexml.rb

```diff
@@ -28,15 +29,17 @@ def self.parse(xml, options)
               temp = stack.pop
               stack.last.add_node(temp)
             end
-          when :text, :cdata
+          when :text
             stack.last.add_node(event[1]) unless event[1].strip.length == 0 || stack.empty?
+          when :cdata
+            stack.last.add_node(raw_data[1]) unless raw_data[1].strip.length == 0 || stack.empty?
           end
         end
         stack.length > 0 ? stack.pop.to_hash : {}
       end
 
       def self.unnormalize(event)
-        event.map! do |el|
+        event.map do |el|
```

Comment:
Q: Was this change something which this PR depends on?

Reply:
yep.  without that, you'd have to call `dup` somewhere so `raw_data` isn't updated with the normalized string